src/expressions.js

```diff
@@ -223,11 +223,26 @@ var _ = Mavo.Expressions = $.Class({
 				path = [];
 			}
 
+			let ignore;
 			if (node.hasAttribute("mv-expressions-ignore")) {
-				var ignore = new Set(node.getAttribute("mv-expressions-ignore").trim().split(/\s*,\s*/));
+				ignore = new Set(node.getAttribute("mv-expressions-ignore").trim().split(/\s*,\s*/));
 			}
 
-			$$(node.attributes).forEach(attribute => {
+			let attributes = $$(node.attributes);
+
+			// Iterate only over mv-attr-foo with expressions
+			for (let attribute of attributes.filter(attribute => attribute.name.startsWith("mv-attr-") && syntax.test(attribute.value))) {
+				let name = attribute.name.replace("mv-attr-", "");
+				let ignorable = attributes.find(attribute => attribute.name === name);
+
+				if (ignorable) {
+					// Ignore expressions in foo, if any, since mv-attr-foo has priority
+					ignore ??= new Set();
+					ignore.add(ignorable.name);
+				}
+			}
+
+			attributes.forEach(attribute => {
```

Comment:
@LeaVerou, thank you so much for your hint—`expressions.js` is indeed the appropriate place for this logic. Sorry for requesting your review one more time. 🙈

Reply:
I think this may be complicating things a bit. I just pushed a commit that tries to simplify the logic.

Also, we don't need to apply `syntax.test()` here, the values of `mv-attr-*`, like directives, are *always* expressions. If you haven't designed it like that, you should.

Reply:
I re-designed the `mv-attr-*` feature to work as `mv-value`. Because of the wrong design decision, I wrote way too complicated logic. I knew that `mv-attr-foo` doesn't make sense if it's not an expression, but I chose the wrong path to implement it. Thank you for pointing this out. 🙏🏻